### DISCUSSION TITLE
Aurora no longer works with Alienware Laptops

### DISCUSSION THREAD

**Discussion by Billybobertjoe** [T-Troll/alienfx-tools#74] [Show and tell]
it says in the aroura devices and wrappers tab that light FX is initialised, but my keyboard and mouse lights dont change. i use an alienware M15 R4 and an alienware AW610M mouse

**Comment by T-Troll** [comment]
It can be AWCC bug. Try to downgrade AWCC to older version or just play with LightFX.dll component.

**Comment by Billybobertjoe** [comment]
what version should i downgrade to?

**Comment by T-Troll** [comment]
Any older ("Previous"). Maybe something before May, 2020 will be ok.

By the way - I can't fix Aurora - i'm not the Aurora developer. I can only add you mouse support into AlienFX-Tools, and it's SDK.
Aurora using Dell one, not mine for now, AFAIK.
So if you want the tool to control lights outside AWCC - yes, we can do. But if you're looking for Aurora support - i'm not the right person for it!
